ballerina/types.bal

```diff
@@ -24,12 +27,19 @@ public type QueueManagerConfiguration record {|
 |};
 
 public enum ConnectionOpenOptions {
-    MQOO_OUTPUT = "MQOO_OUTPUT",
+    // MQOO_OUTPUT = "MQOO_OUTPUT",
```

Comment:
Why have we commented this out ?

Reply:
I think instead of an enum, we will have to create another type for queue access as well. Commented this for now as there is also a type defined as `MQOO_OUTPUT`